core/components/organisms/listbox/nestedList/NestedList.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { menuItemAnimation, getAnimationClass } from './Animation';
+
+export interface NestedListProp {
+  expanded?: boolean;
+  nestedRow?: React.ReactNode;
```

Comment:
@anuradha9712  Please rename the prop to be `nestedBody` or anything more generic.